tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -27,6 +29,7 @@ import (
     "../wski18n"
 
     "github.com/fatih/color"
+    "github.com/ghodss/yaml"
```

Comment:
@mdeuser wskdeploy parses yaml files, it would be good to double check we use the same since eventually wskdeploy will be integrated into wsk cli